hw1/src/main/java/com/example/hw1/MovieRating.java

```diff
@@ -0,0 +1,87 @@
+package com.example.hw1;
+
+import org.hibernate.annotations.CreationTimestamp;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.Date;
+
+@Entity
+public class MovieRating {
+    @Id
+    @GeneratedValue
+    private long id;
+
+    @NotNull
+    @Size(min=3, max=50)
+    private String title;
+
+    @NotNull
+    @Min(1)
+    @Max(5)
+    private int rating = 1;
+
+    @NotNull
+    @Size(min=3, max=25)
+    private String name;
+
+    @CreationTimestamp
+    public Date date;
+
```

Comment:
This is a pretty clean approach for date. We probably would never set it to anything other than when it was created. I do not know how it is stored (format wise) and I do not see a format used for displaying, this may either cause issues later. (unsure since i have not used this method)

Reply:
Thanks Carlos! So far so good but I'll keep an eye out.